README.md

```diff
@@ -1,10 +1,72 @@
 # Democrafy cli
+This is one of the user applications of the Democrafy System. The Democrafy-cli lets you interact with the Democrafy server by requesting data 
+of the current song and requesting that the song is skipped.This is the only application you have to 
+download being a normal user. 
 
-##### Cliente-CLI
-El Cliente-CLI es un sistema que permite votar y visualizar información básica de la canción que se está reproduciendo.
-Actualmente es solo compatible con Linux.
+## Democrafy system
+The Democrafy System allows a group a people to listen to music that the whole group enjoy. An administrator user will 
```

Comment:
...that the whole group  **would** enjoy...

Reply:
La cambie por 

> The Democrafy System allows a group a people to listen to music that the whole group can enjoy.

La idea es que Todos puedan disfrutar.